app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,27 @@
+class InvitationsController < ApplicationController
+  def new
+    @invitation = Invitation.new
+  end
+
+  def create
+    user = User.find_by(inv_params)
+    event_id = cookies[:event_id]
+    cookies[:event_id] = nil
+    if user
+      @invitation = Invitation.new
+      @invitation.event_id = event_id
+      @invitation.attendee_id = user.id
+      @invitation.save
```

Comment:
- [ ] Instead of using so many lines in Invitations#create include the attributes inside the #new. 
Ex: `Invitation.new(event_id: id1, attendee_id: id2)`